{{cookiecutter.project_slug}}/k8s/prod/kustomization.yaml

```
@@ -1,31 +1,52 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-namespace: {{ cookiecutter.project_dash }}
+namespace: {{ cookiecutter.project_dash }}-prod
 
-bases:
- - ../base
+resources:
+  - ../base
+  - certificate.yaml
+  - cluster-issuer.yaml
+  - ingress-route.yaml
+  - postgres.cnpg.yaml
+  - secrets.yaml
 
 patches:
-- path: patches.yaml
-- patch: |
-    $patch: delete
-    kind: Service
-    metadata:
-      name: mailhog
+- patch: |-
+    - op: replace
+      path: /spec/template/spec/containers/0/command
+      value: ["daphne"]
+    - op: replace
+      path: /spec/template/spec/containers/0/args
+      value: ["--bind", "0.0.0.0", "--port", "8000", "--ping-interval", "15", "--ping-timeout", "5", "config.asgi:application"]
+    - op: add
+      path: /spec/template/spec/imagePullSecrets
+      value: 
+        - name: regcred
+    - op: replace
+      path: /spec/template/spec/initContainers/0/command
+      value: ["sh", "-c", "until pg_isready -h postgres-r -p 5432; do echo waiting for postgres; sleep 2; done;"]
```

Hardcoding the host has bitten me in the past. Can you change this to:
```suggestion
      value: ["sh", "-c", "until pg_isready -h $(POSTGRES_HOST); do echo waiting for postgres; sleep 2; done;"]
```
We can omit the port since 5432 is the default. We need to add a POSTGRES_HOST as an environment variable.

---

Pushed an update.